packages/dev/src/main.ts

```diff
@@ -71,6 +71,14 @@ export interface Features {
    */
   images?: {
     enabled?: boolean
+
+    /**
+     * List of allowed domains for remote images. This list will be merged with
+     * any domains coming from the `remote_images` configuration property.
+     *
+     * {@link} https://docs.netlify.com/image-cdn/overview/#remote-path
+     */
+    remoteImages?: string[]
```

Comment:
Part of this is existing docs/naming confusion (IMO), but I wonder if we can avoid spreading some of this.

`remoteImages` doesn't express what the array should contain, and "list of allowed domains" implies it's a list of allowed domains, when it's actually (AFAIK) a list of fully permissive regular expressions as strings that match on the full requested image URL.